### ISSUE TITLE
Instructions for running examples

### ISSUE THREAD

**Issue by White-Rabbit-Scientific** [amPerl/egui-phosphor#21]
Please add instructions for running the examples.

cargo run --release --features="thin light regular bold fill" --example multiple_variants

**Comment by bircni** [comment]
This is already fixed in a PR #20

**Comment by amPerl** [comment]
running an example is part of cargo's documentation. as far as the required features go, it tells you when you try to run it
```
❯ cargo run --example multiple_variants
error: target `multiple_variants` in package `egui-phosphor` requires the features: `thin`, `light`, `regular`, `bold`, `fill`   
Consider enabling them by passing, e.g., `--features="thin light regular bold fill"`
```
